proposals/stdlib/mutablelist-move.md

```diff
@@ -0,0 +1,145 @@
+# Adding moveXXX() Functions to MutableList
+
+It would be helpful to create some additional extension functions for `MutableList` to move items to new indexes, as well as move them up/down, and move one or all items that meet a predicate.
+
+
+# Title
+
+* **Type**: Standard Library API proposal
+* **Author**: Thomas Nield
+* **Contributors**: N/A
+* **Status**: Submitted
+* **Prototype**: Implemented
+
+
+## Summary
+
+It would be helpful to create some additional extension functions for `MutableList` to move items to new indexes, as well as move them up/down, and one or all items that meet a predicate.
+
+These can be implemented using only extension functions on `MutableList`.
+
+## Similar API review
+
+Cannot think of an explicit precendent in another API.
```

Comment:
I figured there was something out there. Not surprised the `Collections` helper static class has a method, although I've never used it. I will investigate those.